File: tests/fixtures/typescript-library/src/models/genre.ts
/** Genre categories for library books. */
export enum Genre {
    Fiction = 'fiction',
    NonFiction = 'non_fiction',
    Science = 'science',
    History = 'history',
    Biography = 'biography',
}

/** Human-readable label for a genre. */
export function genreLabel(genre: Genre): string {
    return genre.replace('_', ' ');
}

File: tests/fixtures/java-library/src/main/java/library/models/Genre.java
package library.models;

/** Genre categories for library books. */
public enum Genre {
    FICTION,
    NON_FICTION,
    SCIENCE,
    HISTORY,
    BIOGRAPHY;

    /** Human-readable label for display. */
    public String label() {
        return name().replace("_", " ").substring(0, 1)
            + name().replace("_", " ").substring(1).toLowerCase();
    }
}
